Library/Formula/libpng.rb

```diff
@@ -28,6 +28,7 @@ class Libpng < Formula
 
   def install
     ENV.universal_binary if build.universal?
+    system "autoreconf", "--force", "--install" if head?
```

Comment:
The shorter `-fvi` command is used frequently throughout Homebrew where required, so it's nice to stick to the standard there.